src/main/java/cj/software/genetics/schedule/javafx/SchedulingController.java

```diff
@@ -98,26 +130,73 @@ public void newProblem() {
         Optional<ProblemSetup> optProblemSetup = newProblemDialog.showAndWait();
         if (optProblemSetup.isPresent()) {
             logger.info("a new problem was defined");
-            ProblemSetup problemSetup = optProblemSetup.get();
+            problemSetup = optProblemSetup.get();
             List<Task> allTasks = createAllTasks(problemSetup);
             List<Solution> allSolutions = solutionService.createInitialPopulation(
                     problemSetup.getNumSolutions(),
                     problemSetup.getNumWorkers(),
                     problemSetup.getNumSlots(),
                     allTasks);
-            ObservableList<Solution> tableData = FXCollections.observableList(allSolutions);
-            tblSolutions.setItems(tableData);
-            if (!allSolutions.isEmpty()) {
-                tblSolutions.getSelectionModel().select(0);
-            }
+            setPopulation(allSolutions, 0);
```

Comment:
how can you be sure that the `ProblemSetup` object has the value 0? Use its `getCurrentValue()` method instead